Feature/read all costume approval board/costume approval board

hyeneung opened the pull request:
## #️⃣연관된 이슈
- this closes #62 

## 📝작업 내용
- 의상 게시판 목록 조회

### 스크린샷 (선택)
![image](https://github.com/user-attachments/assets/02dcbe74-44b6-439e-b21c-d491683671ee)
![image](https://github.com/user-attachments/assets/988a33ca-abf9-4829-94c4-994d7020c2c1)

hyeneung committed:
style: 현재 브랜치 작업 위한 수정

hyeneung committed:
feature: 목록 조회 response dto 작성

hyeneung committed:
feature: 접근 권한 예외 처리

hyeneung committed:
feature: repository 필요 메서드 작성

hyeneung committed:
feature: 목록 조회 기능 정의 및 구현

hyeneung committed:
feature: 목록 조회 controller 작성

hyeneung committed:
style: pull 중 merge conflict 해결

Merge branch 'feature/read-all-costume-approval-board/costume-approval-board' of https://github.com/pangseon/Extra into read-all-company
# Conflicts:
#	src/main/java/com/example/extra/domain/costumeapprovalboard/CostumeApprovalBoard/CostumeApprovalBoardController.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/entity/CostumeApprovalBoard.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/exception/CostumeApprovalBoardErrorCode.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/repository/CostumeApprovalBoardRepository.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/service/CostumeApprovalBoardService.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/service/impl/CostumeApprovalBoardServiceImpl.java

shin0112 commented:
company 관련해서 리팩토링한 게 전부 롤백 되서요!
로직이랑은 관련 없고 url이랑 필드 네임만 변경된거라서.. 이 부분 좀 확인해주실 수 있나요?

shin0112 commented on src/main/java/com/example/extra/domain/applicationrequest/exception/ApplicationRequestErrorCode.java:
삭제된 error code 확인 부탁드립니다

hyeneung replied:
넵 고쳐서 다시 커밋 해놓겠습니다

hyeneung committed:
Merge branch 'develop' of https://github.com/pangseon/Extra into feature/read-all

hyeneung committed:
Revert "style: pull 중 merge conflict 해결"

This reverts commit 4aa4bcb165e9fc9d49e8e6e417c8e45606019024, reversing
changes made to b16c90ff64acf86fc036f6212790e55ebb805ca8.

hyeneung committed:
style: stash 로 저장해둔 file 변경점 적용

shin0112 reviewed (approved):
수고하셨습니다~ 😆

hyeneung committed:
style : merge conflict 해결

Merge remote-tracking branch 'origin/develop' into feature/read-all-costume-approval-board/costume-approval-board
# Conflicts:
#	src/main/java/com/example/extra/domain/costumeapprovalboard/controller/CostumeApprovalBoardController.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/entity/CostumeApprovalBoard.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/exception/CostumeApprovalBoardErrorCode.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/repository/CostumeApprovalBoardRepository.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/service/CostumeApprovalBoardService.java
#	src/main/java/com/example/extra/domain/costumeapprovalboard/service/impl/CostumeApprovalBoardServiceImpl.java
